cmd/controller/keys.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"errors"
+	"io"
+	"math/big"
+	"time"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	certUtil "k8s.io/client-go/util/cert"
+)
+
+const SealedSecretsKeyLabel = "sealed-secrets-key"
```

Comment:
This should be namespaced to avoid collisions with local values / other controllers: `sealedsecrets.bitnami.com/sealed-secrets-key`